deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dhstore
+spec:
+  selector:
+    matchLabels:
+      app: dhstore-helga
+  template:
+    metadata:
+      labels:
+        app: dhstore-helga
+    spec:
+      topologySpreadConstraints:
+        - maxSkew: 1
+          topologyKey: topology.kubernetes.io/zone
+          whenUnsatisfiable: ScheduleAnyway
+      containers:
+        - name: dhstore-helga
+          args:
+            - '--storePath=/data'
+            - '--disableWAL'
+            - '--blockCacheSize=2Gi'
+          volumeMounts:
+            - name: data
+              mountPath: /data
+          resources:
+            limits:
+              cpu: "6"
+              memory: 60Gi
+            requests:
+              cpu: "6"
+              memory: 60Gi
+          ports:
+            - containerPort: 40081
+              name: metrics
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: node.kubernetes.io/instance-type
+                    operator: In
+                    values:
+                      - r5n.2xlarge
+                  - key: topology.kubernetes.io/zone
+                    operator: In
+                    values:
+                      - us-east-2c    
```

Comment:
Checking this is different from the current AZ used by the existing dhstore?

Reply:
I think it's the same. Should it be different?